src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+    private final Random random = new Random();
+
+    public Ball getRandomBall() {
+        return new Ball(colorSupplier.getRandomColor(), random.nextInt(100));
```

Comment:
Create constant for 100
```suggestion
        return new Ball(colorSupplier.getRandomColor(), random.nextInt(MAX_NUMBER));
```

Reply:
Checkstyle не дозволяє створити MAX_NUMBER великими літерами